src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/001_113_aggiuntaNuovaDelegaPG.feature

```diff
@@ -14,9 +14,10 @@ Feature: La persona giuridica aggiunge una nuova delega
     And Si controlla che non sia presente una delega con stesso nome "nuovaDelegaPG" persona giuridica
     And Nella sezione Delegati dell impresa click sul bottone aggiungi nuova delega
     And Si visualizza la sezione Aggiungi Delega persona giuridica
-    And Nella sezione Aggiungi Delega persona giuridica inserire i dati "nuovaDelegaPG"
     And Nella sezione Aggiungi Delega persona giuridica verificare che la data sia corretta
```

Comment:
In this case, please disregard this check, as I have provided a better solution above.